netlify-templates/nuxt-toolbox
# Tn/add cypress

# Summary

Added Cypress to help with template maintenance. With this template having a feedback form, I've created a few tests outlined below that go through the process of testing out the validation and form submission.

## Tests

- check for validation message for invalid email input
- check for validation message for no email input
- check for validation message with email and no feedback
- submitting feedback should redirect to /success
